Resources/Prototypes/Entities/Structures/Wallmounts/Signs/posters.yml

```diff
@@ -848,7 +848,7 @@
   parent: PosterBase
   id: PosterLegitEnlist
   name: "Enlist"
-  description: "Enlist in the Nanotrasen Deathsquadron reserves today!"
+  description: "Enlist in the NanoTrasen ERT reserves today!"
```

Comment:
I suggest adding a comment to specify that we changed this basegame file
`# DeltaV - Removed mention of Deathsquad` or something